src/content/en/updates/2017/06/play-request-was-interrupted.md

```diff
@@ -0,0 +1,146 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What is really happening with "DOMException: The play() request was interrupted"?
+
+{# wf_updated_on: 2017-06-13 #}
+{# wf_published_on: 2017-06-13 #}
+{# wf_tags: media,devtools #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: What is really happening with "DOMException: The play() request was interrupted"? #}
+
+# DOMException: The play() request was interrupted {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Did you just stumble upon this unexpected media error in the Chrome DevTools
+JavaScript Console?
+
+<blockquote class="warning">
+Uncaught (in promise) DOMException: The play() request was interrupted by a new
+load request.
+</blockquote>
+
+<blockquote class="warning">
+Uncaught (in promise) DOMException: The play() request was interrupted by a
+call to pause().
+</blockquote>
+
+You're in the right place then. Have no fear. I'll explain [what is causing
+this](#error) and [how to fix it](#fix).
+
+## What is causing this {: #error }
+
+Here's some JavaScript code below that reproduces the "uncaught (in promise)"
+error you're seeing:
+
+<span class="compare-worse">DON'T</span>
+```html
+<video id="video" preload="none" src="https://example.com/file.mp4"></video>
+
+<script>
+  video.play(); // <-- WRONG! This is NOT synchronous.
+  video.pause();
+</script>
+```
+
+The code above results in this error message in Chrome DevTools: 
+
+> _Uncaught (in promise) DOMException: The play() request was interrupted by a
+> call to pause()._
+
+This is due to the fact that the `play()` call is NOT synchronous, like it used
+to be. Since Chrome 50 - [March 2016], a `play()` call on an a `<video>` or
+`<audio>` element returns a [Promise], a function that returns a single result
+**asynchronously**. If playback succeeds, the Promise is fulfilled, and if
+playback fails, the Promise is rejected along with an error message explaining
+the failure.
+
+As `video.play()` is asynchronous, `video.pause()` is executed before
+video playback actually starts. This causes `video.play()` to return a rejected
+Promise as user just asked to pause video. Think about it this way:
+`video.play()` fails eventually.
+
+Note: Calling `video.pause()` isn't the only way to reset video playback state.
+It also occurs with `video.src = ''`, `video.removeAttribute('src')`, and
+`video.load()` for instance.
```

Comment:
Also, `remoteAttribute('src')` is a no-op I believe. I don't think it loads anything.